Resources/keybinds.yml

```diff
@@ -187,6 +187,10 @@ binds:
 - function: OpenLanguageMenu
   type: State
   key: L
+- function: OpenConsentWindow
+  type: State
+  key: C
```

Comment:
Key "C" is already used, you need to change this.

Reply:
It's actually Alt+C see one line below

Reply:
Correct but during testing, this reported an error, non-fetal error tho.

Reply:
I don't think we really need a hotkey for this. I doubt people will need to change their consent quickly.